ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -147,11 +147,11 @@ public KsqlRestConfig(Map<?, ?> props) {
     return getPropertiesWithOverrides(KSQL_STREAMS_PREFIX);
   }
 
-  public String getCommandTopic() {
+  public String getCommandTopic(String ksqlServiceId) {
     return String.format(
-        "%s_%s",
-        KsqlConfig.KSQL_SERVICE_ID_DEFAULT,
-        getString(COMMAND_TOPIC_SUFFIX_CONFIG)
```

Comment:
I don't think the COMMAND_TOPIC_SUFFIX_CONFIG constants are used anywhere anymore. Lets clean them up?